src/components/network-map-tab.js

```diff
@@ -522,18 +549,29 @@ export const NetworkMapTab = ({
                   }
               );
 
+        const nodeBeforeFetch = currentNodeRef.current;
         Promise.all([substationPositionsDone, linePositionsDone])
             .then(() => {
                 temporaryGeoDataIdsRef.current = new Set();
             })
             .catch(function (error) {
                 console.error(error.message);
-                setErrorMessage(
-                    intlRef.current.formatMessage(
-                        { id: 'geoDataLoadingFail' },
-                        { studyUuid: studyUuid }
-                    )
-                );
+
+                // we display the error to the user only if the node is built
+                if (
+                    isSameNodeAndBuilt(currentNodeRef.current, nodeBeforeFetch)
```

Comment:
Add this `isSameNodeAndBuilt` check in the `fetchLinePositions.then` and `fetchSubstationPositions.then` as well

Reply:
done